.github/workflows/verifier-e2e.all.workflow_dispatch.main.all.slsa3.yml

```diff
@@ -81,6 +83,78 @@ jobs:
       base64-subjects: "${{ needs.build.outputs.digest }}"
       compile-generator: true
 
+  gha_go:
+    permissions:
+      id-token: write # For signing.
+      contents: write # For asset uploads.
+      actions: read # For the entrypoint.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@v1.2.2
+    with:
+      go-version: 1.18
+      # We cannot use ${{ env.CONFIG_FILE }} because env variables are not available.
```

Comment:
It's a bit of duplication but, you could maybe make two different jobs for push and workflow_dispatch that have `if: github.event_name == 'workflow_dispatch'` and `if: github.event_name == 'push'` and have them use `config-ldflags-workflow_dispatch-tag-name.yml` and `config-ldflags-push-tag-name.yml` respectively.

This method might also be valid for the other jobs.